sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec

```diff
@@ -12,7 +12,7 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio and/or the SonarQube platform.</summary>
     <description>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio (http://vs.sonarlint.org) and/or the SonarQube platform (http://www.sonarqube.org).</description>
-    <releaseNotes>http://www.sonarlint.org/visualstudio/rules/index.html#version=6.5.0.0</releaseNotes>
+    <releaseNotes>http://www.sonarlint.org/visualstudio/rules/index.html#version=6.6.0.0</releaseNotes>
```

Comment:
The link is no longer valid on redesigned website. Check with marketing/Christophe where the release notes go these days.

Reply:
Let's still merge like this (as this PR is only about bumping the version). Let's create a ticket and fix it right after.

Reply:
Ticket: https://github.com/SonarSource/sonar-csharp/issues/858